src/index.js

```diff
@@ -40,12 +40,18 @@ app.use(
 );
 
 app.keys = (process.env.COOKIE_SECRETS || '').split(',');
+app.proxy = !!process.env.TRUST_PROXY_HEADERS;
```

Comment:
should we add trust proxy check? 
https://github.com/SagePtr/koa-trust-proxy

Reply:
Sounds cool!
But what is the IP address of nginx, as perceived by rumors-api container?